docs/source/algorithms/ConvertToMD-v1.rst

```diff
@@ -281,6 +288,35 @@ This example produces 3-dimensional dataset, with a temperature axis.
 
    Resulting MD workspace contains 605880 events and 3 dimensions
 
+**Example - Convert to Q-space, indexed version:**
+
+The code iteslef works but disabled from doc tests as takes too long to complete. User should provide its own
```

Comment:
typo: iteslelf -> itself.

typo: User should provide its own -> User should provide their own

Just an idea - you've done well in some of the system tests using `CreateSampleWorkspace` since this elides the `Load` and possibly gives fewer events, would this then make the doc-test possible?

Reply:
1. It is not really disabled, I've copied this from FindPeaksMD and have already removed. 2.CreateSampleWorkspace creates wired events set, I want the user who just copypaste this example in mantid script window has a reliable picture. So I prefer to use the real file, I tales ~15 seconds overall.